README.md

```diff
@@ -203,6 +207,7 @@ Lastly, you can use `--help` on `swiftgen` or one of its subcommand to see the d
 While we highly recommend the use a configuration file for performance reasons (especially if you have multiple outputs, but also because it's more flexible), it's also possible to directly invoke the available subcommands to parse various resource types:
 
 * `swiftgen colors [OPTIONS] FILE1 …`
+* `swiftgen coredata [OPTIONS] FILE1 …`
```

Comment:
Now that all parsers accept directories and not just files, shouldn't we uniformize here the example calls (probably with `DIR1 …` or `DIRorFILE1 …`) everywhere, instead of having some still with `FILE1 …` and some with `DIRORFILE1 …`…?